src/main/java/core/basesyntax/JavaDateTimeApi.java

```diff
@@ -1,125 +1,173 @@
 package core.basesyntax;
 
+import java.time.DateTimeException;
+import java.time.Instant;
 import java.time.LocalDate;
 import java.time.LocalDateTime;
 import java.time.LocalTime;
 import java.time.OffsetDateTime;
+import java.time.ZoneId;
+import java.time.ZoneOffset;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+import java.util.Locale;
 import java.util.Optional;
 
 public class JavaDateTimeApi {
+    public static final String TIME_ZONE = "+02:00";
+    public static final String TIME_FORMATTER_FORMAT_DATE = "dd MMMM yyyy HH:mm";
+    public static final String TIME_FORMATTER_CUSTOM_PARSE_DATE = "d MMM yyyy";
+
+    Locale locale = Locale.US;
```

Comment:
You don't need this class variable. Use `Locale.US` in places where you are using `locale`

Reply:
done